src/evergreen/stats.py

```diff
@@ -39,10 +39,10 @@ class TaskStats(_BaseEvergreenObject):
     task_name = evg_attrib("task_name")
```

Comment:
The [TaskStats](https://github.com/evergreen-ci/evergreen/blob/f10690476e6d3b0a69b2d9432f513eb22a7e23bc/model/stats/query.go#L278-L293) response doesn't have a `test_file` field. Should we remove it?

Reply:
Yes, let's remove it.